packages/cactus-plugin-ledger-connector-aries/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ghcr.io/hyperledger/cactus-cmd-api-server:v1.0.0
+
+ARG NPM_PKG_VERSION=latest
+
+RUN npm i @hyperledger/cactus-plugin-ledger-connector-aries@${NPM_PKG_VERSION} --production
```

Comment:
@outSH I highly recommend using a much newer version of the API server container image, something from 2024.
I can help with publishing the package to npm as a canary release if you'd like to test with it.

I'm also working on a pull request that fixes a lot of the issues with the container images security and plugin installation-wise, so if you'd like to collaborate on this closer together just let me know and I can make some changes here myself as well!

Reply:
I don't really need a docker container for this package so I've removed it for now, can be added some other day

Reply:
@outSH Works for me! Thank you!